.github/workflows/cd.yml

```diff
@@ -1,6 +1,5 @@
 name: CD
-
-on:
+"on":
```

Comment:
This style fixes should be done in a dedicated pull request, were this automatically done by running `pre-commit run -a` ?

Reply:
Okay! I have been running `pre-commit run --all-files`

Reply:
I do not remember now if the styling errors prevented me from going further , but I spent a lot of time before running pre-commit locally and the gha runs would fail even on small errors.

Reply:
Is it correct that `on` is in double-quotes: `"on"`?

Reply:
I think as JC mentioned.. some of the styling changes are automatically fixed by running the `pre-commit run --all-files` command. As evident from successful CI/CD runs, I think it may be okay to have double quotes

Reply:
While these changes may be valid, I do not understand where they come from.

Two observations:

1. The CI is that ran ~5 hours ago complete[^1] without these.

2. After running pre-commit locally there are not applied:

    ```
    git clone https://github.com/ImagingDataCommons/idc-index-data
    cd idc-index-data
    workon pre-commit # virtual env I used to run pre-commit
    pip install --upgrade pre-commit
    pre-commit run --all-files
    ```

I then suggest we do not apply these changes, they introduce noise in the history and are not needed to implement the feature.

[^1]: https://github.com/ImagingDataCommons/idc-index-data/actions/runs/8392774664